backend/apps/accounts/tests/test_dump_anonymized_db.py

```
@@ -0,0 +1,129 @@
+        # Second user for deterministic test comparison
+        self.user2 = User.objects.create_user(
+            username="real_janedoe",
+            email="jane@example.com",
+            password="realpassword123",
+            first_name="Jane",
+            last_name="Doe",
+        )
+
+    def _run_command(self, *args, **kwargs):
+        out = StringIO()
+        call_command("dump_anonymized_db", stdout=out, *args, **kwargs)
```

Ruff (0.15.20)

[warning] Star-arg unpacking after a keyword argument is strongly discouraged

(B026)